packages/contracts/evm-contracts/contracts/token/IUri.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+/// @dev An interface exposing the `uri` function from IERC1155MetadataURI.
+interface IUri {
+    /**
+     * @dev Returns the URI for token type `id`.
+     *
+     * If the `\{id\}` substring is present in the URI, it must be replaced by
+     * clients with the actual token type ID.
+     */
+    function uri(uint256 id) external view returns (string memory);
```

Comment:
What's the reason for using this instead of `IERC1155MetadataURI` directly?

Reply:
`IERC1155MetadataURI` inherits `IERC1155`, so using it would dictate all requirements of `IERC1155`, which are not needed in `uri(uint256 id, IUri customUriInterface)` function call.